Services/Masterdata/Services.Masterdata.EF/Services/MasterdataService.cs

```diff
@@ -0,0 +1,164 @@
+﻿using AutoMapper.QueryableExtensions;
+using Lens.Core.Lib.Exceptions;
+using Lens.Core.Lib.Services;
+using Lens.Services.Masterdata.EF.Entities;
+using Lens.Services.Masterdata.Models;
+using Lens.Services.Masterdata.Services;
+using Microsoft.EntityFrameworkCore;
+using System.Linq.Expressions;
+
+namespace Lens.Services.Masterdata.EF.Services;
+
+public class MasterdataService : BaseService<MasterdataService>, IMasterdataService
+{
+    private readonly MasterdataDbContext _masterdataDbContext;
+
+    public MasterdataService(MasterdataDbContext masterdataDbContext, IApplicationService<MasterdataService> applicationService) : base(applicationService)
+    {
+        _masterdataDbContext = masterdataDbContext;
+    }
+
+    public async Task<IEnumerable<MasterdataTypeListBM>> GetMasterdataTypes()
```

Comment:
For get all I suggest to return the ResultListModel < MasterdataTypeListModel > with size/value. It might be useful for FE app.

Reply:
Changed this.